homeassistant/components/ecobee/climate.py

```diff
@@ -726,6 +740,77 @@ def resume_program(self, resume_all):
         )
         self.update_without_throttle = True
 
+    def set_active_sensors(
```

Comment:
Naming should be rethought, 

Active Sensor makes it sounds like it is a property of the sensor, but in fact it's a property of the schedule/comfort settings.

Reply:
Maybe `set_participating_sensors`? That would be closer to what ecobee calls the functionality. I am open to changing it to whatever makes the most sense.

Reply:
I like that better 

other options

`set_sensors_used_in_climate`